build.gradle

```diff
@@ -234,12 +240,6 @@ subprojects {
 		archives javadocJar, sourcesJar
 	}
 
-	javadoc {
```

Comment:
Minimum JDK version required for building this project is JDK 11, html5 is default.